Classes/Backend/TceMain.php

```diff
@@ -149,16 +149,20 @@ public function processCmdmap_beforeStart(&$reference)
      * @param string $id The records id (if any)
      * @param string $relativeTo Filled if command is relative to another element
      * @param DataHandler $reference Reference to the parent object (TCEmain)
+     * @param array $pasteUpdate Extended paste command
      * @return void
      */
-    public function processCmdmap_preProcess(&$command, $table, $id, &$relativeTo, &$reference)
+    public function processCmdmap_preProcess(&$command, $table, $id, &$relativeTo, &$reference, $pasteUpdate)
```

Comment:
After taking a look into the TYPO3 source code, I think we may be able to move the "postProcess" logic to "preProcess" and skip record updates, instead receiving $pasteUpdate by reference and filling it with the target information. I'll look into that!